src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Remove this file from the change.

Reply:
I've reverted the file to a version before the encoding got changed, hopefully that removes it from the changeset.